scripts/jestSetup.js

```diff
@@ -1,3 +1,8 @@
+// As of Jest 24 (https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#breaking-changes)
+// this is no longer included by default for async tests
+// So we're including it so we can use async/await
+require('regenerator-runtime/runtime');
```

Comment:
Should we also be polyfilling this in our build? 🤔

Reply:
We're not using async/await in our code anymore. Probably should have some eslint rule to forbid it so we don't have to worry about it.

If we do transpile it down though I do think it should be an assumption of the application to provide their own, and we just make note of that.

Reply:
Ahh wait, this is just for writing async functions in tests. Disregard 😅